scripts/download_logs.sh

```diff
@@ -21,7 +21,7 @@ function download_service_logs() {
     podman ps -a || true
 
     for service in "assisted-service" "assisted-image-service" "assisted-installer-ui" "postgres"; do
-      podman logs ${service} >${LOGS_DEST}/onprem_${service}.log || true
+      podman logs ${service} >${LOGS_DEST}/k8s_${service}_onprem.log || true
```

Comment:
it will be misleading, because this deployment method has nothing to do with kubernetes, no?

Reply:
Right.. I think I didn't thought all the way through this.
I wanted to make the logs with the same prefix as the minikube deployments.
I'll change them both to start with a new prefix